### DISCUSSION TITLE
WinMerge hangs when comparing files through context menu or SendTo

### DISCUSSION THREAD

**Discussion by rpratesp** [WinMerge/winmerge#946] [Q&A]
When I compare 02 files through Windows context menu, Winmerge hangs. The same issue happens if i used a link in "Send To".

**Comment by sdottaka** [comment]
This is probably the same problem as the following issue that was recently fixed

https://github.com/WinMerge/winmerge/issues/913

You can download the latest executable or installer from the following URL

https://github.com/WinMerge/winmerge/suites/3641797390/artifacts/88278439

or

https://ci.appveyor.com/project/sdottaka/winmerge/build/artifacts

**Reply by rpratesp** [reply]
Sorry, but doesnt't work...
![Tela20210901133900](https://user-images.githubusercontent.com/36540230/131710370-923280f0-7cdb-4fbf-bc7f-07ffb4ad9d7b.jpg)
![Tela20210901134208](https://user-images.githubusercontent.com/36540230/131710434-8a10d4e7-a6cf-449c-9733-e1f5dbf16e87.jpg)

**Comment by rpratesp** [comment]
A new information: I had this problem on my machine at home and my machine at work. The error information I passed above was from my home machine. I tested it now on my work machine and the problem was solved with this version. I'll run more tests on my home machine and give you news.
